app/forms/scim_clients/form.rb

```diff
@@ -79,7 +79,7 @@ class Form < ApplicationForm
           name: :jwt_sub,
           label: ScimClient.human_attribute_name(:jwt_sub),
           required: true,
-          caption: I18n.t("admin.scim_clients.form.jwt_sub_description_html", docs_url: "#").html_safe, # TODO: correct docs url
+          caption: I18n.t("admin.scim_clients.form.jwt_sub_description_html", docs_url: ::OpenProject::Static::Links[:sysadmin_docs][:scim_jwt_authetication_method][:href]).html_safe,
```

Comment:
:yellow_circle: Something that I just noticed now: There is a helper to access links that you can use here and for all other references to docs:

```suggestion
          caption: I18n.t("admin.scim_clients.form.jwt_sub_description_html", docs_url: ::OpenProject::Static::Links.url_for(:sysadmin_docs, :scim_jwt_authetication_method)).html_safe,
```